### PR TITLE
Landing page

### LINKED ISSUE #77: Landing Page
When the user hits Classroom for the first time, this is what they currently see. I'm not entirely sure what to do with this, but it could possibly be a page to show off the highlights of the app.

screen shot 2015-08-05 at 12 39 06

### PR DESCRIPTION
**Pull request by tarebyte**
Closes #77 

/cc @johndbritton for copy

### CODE REVIEW

**Comment by tarebyte** [comment]
screen shot 2015-09-15 at 07 03 21

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1 +1,44 @@
...
+    <ul>
+      <li><%= mega_octicon('book')%></li>
+      <li><%= mega_octicon('pencil')%></li>
+      <li><%= mega_octicon('mortar-board')%></li>
+      <li><%= mega_octicon('organization')%></li>
+      <li><%= mega_octicon('calendar')%></li>
+      <li><%= mega_octicon('comment-discussion')%></li>
+    </ul>
+
+    <h1 class="heading">Your course assignments on GitHub</h1>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: This is :sparkles:

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1 +1,44 @@
...
+    <p class="subheading">Automate repository creation and access control, making it easy to distribute starter code and collect assignment submissions on GitHub.</p>
+
+    <h2><%= link_to 'Sign in with your GitHub account to get started', login_path %></h2>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: Can we make this a green or blue action button?

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1 +1,44 @@
...
+      <li><%= mega_octicon('book')%></li>
+      <li><%= mega_octicon('pencil')%></li>
+      <li><%= mega_octicon('mortar-board')%></li>
+      <li><%= mega_octicon('organization')%></li>
+      <li><%= mega_octicon('calendar')%></li>
+      <li><%= mega_octicon('comment-discussion')%></li>
+    </ul>
+
+    <h1 class="heading">Your course assignments on GitHub</h1>
+    <p class="subheading">Automate repository creation and access control, making it easy to distribute starter code and collect assignment submissions on GitHub.</p>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: Let's use "Classroom for GitHub automates repository creation and access control, making it easy to distribute starter code and collect assignments on GitHub."

We can probably do better, but what's currently there doesn't seem correct.

**Comment by johndbritton** [comment]
In the feature explainers on the bottom of the page you can drop words like you/your/their.

* Create assignments
* Invite students
* Review progress

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -1 +1,44 @@
...
+
+    <h2><%= link_to 'Sign in with your GitHub account to get started', login_path %></h2>
+  </div>
+</div>
+
+<div class="information-section">
+  <div class="container">
+    <div class="columns">
+      <div class="one-third column">
+        <h2>Create your assignments</h2>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: "Create assignments"

File: app/views/pages/home.html.erb [hunk 5]
```diff
@@ -1 +1,44 @@
...
+        <p>
+          You have the ability to create both individual or group assignments, that can be public or private on GitHub.
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: Students work on assignments individually or in groups using public or private GitHub repositories.

File: app/views/pages/home.html.erb [hunk 6]
```diff
@@ -1 +1,44 @@
...
+        </p>
+      </div>
+
+      <div class="one-third column">
+        <h2>Invite your students</h2>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: "Invite students"

File: app/views/pages/home.html.erb [hunk 7]
```diff
@@ -1 +1,44 @@
...
+        <p>
+          Each assigment comes with a unique invitation url, so adding students to an organization is much easier.
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: Students gain access to assignments using unique invitation URLs, eliminating the need to collect GitHub usernames in advance and manage access manually.

File: app/views/pages/home.html.erb [hunk 8]
```diff
@@ -1 +1,44 @@
...
+        </p>
+      </div>
+
+      <div class="one-third column">
+        <h2>View their progress</h2>
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: "View progress"

File: app/views/pages/home.html.erb [hunk 9]
```diff
@@ -1 +1,44 @@
...
+        <p>
+          With GitHub you can view their progress every step of the way.
```
Comment (johndbritton) [inline app/views/pages/home.html.erb]: Teachers and teaching assistants can easily browse submissions by assignment, making it easy to follow along with student progress.

**Comment by tarebyte** [comment]
> Can we make this a green or blue action button?

screen shot 2015-09-15 at 12 04 58

@johndbritton @jglovier what do you think about this button?

**Comment by johndbritton** [comment]
Button looks good to me, not sure if there's a standard style that should be used but I'm happy to :ship: with this.

Can we update the text under "Create assignments", the fact that it's not four lines is bugging me.

> Distribute assignments with optional starter code. Students work on assignments individually or in groups using public or private GitHub repositories.

**Comment by tarebyte** [comment]
:shipit:

**Comment by jglovier** [comment]
>what do you think about this button?

Can you use the same blue gradient style button as you see on https://education.github.com/'s "Student Developer Pack" section?

**Comment by tarebyte** [comment]
@jglovier I sure can

**Comment by jglovier** [comment]
@tarebyte :heart: :metal:

**Pull request merged**